res/css/views/rooms/_EventTile.scss

```diff
@@ -258,17 +258,11 @@ $left-gutter: 64px;
     display: inline-block;
     width: 14px;
     height: 14px;
-    top: 29px;
+    top: -20px;
```

Comment:
can we get a comment for where -20 comes from?

Reply:
Thanks for asking, made me realize that using pixels may not be a good idea...